src/main/kotlin/org/rust/lang/core/resolve/NameResolution.kt

```diff
@@ -305,6 +305,10 @@ fun processPathResolveVariants(lookup: ImplLookup, path: RsPath, isCompletion: B
             val superMod = containingMod.`super`
             if (superMod != null && processor("super", superMod)) return true
             if (crateRoot != null && processor("crate", crateRoot)) return true
+            if (path.kind == PathKind.IDENTIFIER &&
+                path.containingCargoTarget?.edition == CargoWorkspace.Edition.EDITION_2018) {
```

Comment:
We already have `target` in scope?

Reply:
Nvm, that target is completely unrelated 
https://github.com/intellij-rust/intellij-rust/pull/2862/files#diff-a2b72fdf4730d3704af99c0f7c4d6103R175 :)

Reply:
Okay 😄